controllers/storagecluster/storageclient.go

```diff
@@ -0,0 +1,63 @@
+package storagecluster
+
+import (
+	"fmt"
+
+	ocsclientv1a1 "github.com/red-hat-storage/ocs-client-operator/api/v1alpha1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	"github.com/red-hat-storage/ocs-operator/v4/services"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	cutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	storageClientName = "local-storageclient"
```

Comment:
The name should match the storage cluster name, use that and add a `-client` suffix to it or something similar.